ui/packages/plot/src/Plot.svelte

```diff
@@ -1,20 +1,55 @@
-<script>
+<script lang='ts'>
 	//@ts-nocheck
 	import Plotly from "plotly.js-dist-min";
 	import { Plot as PlotIcon } from "@gradio/icons";
+	import { colors as color_palette, ordered_colors } from "@gradio/theme";
+	import { get_next_color } from "@gradio/utils";
 	import { Vega } from "svelte-vega";
```

Comment:
We'll probably want to dynamically load the SDK for the specific plot in the future but this is fine for now. Just mentioning it here. No changes required.